src/components/SingleReview.js

```diff
@@ -11,16 +12,13 @@ const SingleReview = () => {
   const { review_id } = useParams();
 
   useEffect(() => {
-      setLoading(true);
-      
-
-      getReviewById(review_id).then((reviewFromApi) => {
-        setReview(reviewFromApi);
-        setLoading(false);
-      });
-    },
-    [review_id]
-  );
+    setLoading(true);
```

Comment:
Same comment as last one, you're setting loading to be true further up this file already